src/assets/scripts/client/aircraft/AircraftModel.js

```diff
@@ -1981,17 +1981,27 @@ export default class AircraftModel {
             holdParameters.inboundHeading = bearingToHoldFix;
         }
 
-        const { inboundHeading } = holdParameters;
+        const { inboundHeading, legLength } = holdParameters;
         const outboundHeading = radians_normalize(inboundHeading + Math.PI);
         const groundTrack = radians_normalize(this.groundTrack);
         const offset = getOffset(this, waypointRelativePosition, inboundHeading);
-        const holdLegDurationInMinutes = holdParameters.legLength.replace('min', '');
-        const holdLegDurationInSeconds = holdLegDurationInMinutes * TIME.ONE_MINUTE_IN_SECONDS;
         const gameTime = TimeKeeper.accumulatedDeltaTime;
         const isPastFix = offset[1] < 1 && offset[2] < 2;
         const isTimerSet = holdParameters.timer !== INVALID_NUMBER;
         const isTimerExpired = isTimerSet && gameTime > holdParameters.timer;
 
+        let holdLegDurationInSeconds;
+        if (legLength.indexOf('min') !== -1) {
```

Comment:
* `let` assignment should immediately follow `const` without an empty line
* missing empty line above `if`

Reply:
Every time. You'd think is learn...